framework/helpers/BaseInflector.php

```diff
@@ -410,6 +410,18 @@ public static function id2camel($id, $separator = '-')
         return str_replace(' ', '', StringHelper::mb_ucwords(str_replace($separator, ' ', $id), self::encoding()));
     }
 
+    /**
+     * Converts a CamelCase action name into an ID in lowercase.
+     * Words in the ID are concatenated using the specified character '-'.
+     * For example, 'CreateUser' will be converted to 'create-user'.
+     * @param string $name the string to be converted
+     * @return string the resulting ID
+     */
+    public static function camel2idAction($name)
```

Comment:
This is very specific method used only in one place - IMO it is better to move it to `HelpController` than clutter generic helper.

Reply:
This method also uses the private method `self::encoding()` copy it to the `HelpController` as well?

Reply:
It is used only for processing method name, IMO we could just hardcode `UTF-8`.